docs/semgrep-pro-vs-oss.md

```diff
@@ -1,207 +1,411 @@
       <dt>Semgrep Secrets</dt><dd>A secrets scanner that, in addition to detecting secrets, validates these leaked secrets on a variety of services to help you prioritize active secrets.</dd>
-      <dt>Semgrep AppSec Platform</dt><dd>A web application for the deployment, management, and monitoring of findings from Semgrep's SAST, SCA, and secrets scanners. It integrates with continuous integration (CI) providers such as GitHub Actions, GitLab CI/CD, CircleCI, and more.</dd>
     </dl>
   </dd>
 </dl>
 
+:::note
+Semgrep Code and Semgrep Supply Chain are free for up to 10 contributors.
+:::
+
+## Comparison by core workflows
+
+![Scope of each offering by core workflows](/img/security-program-workflows.svg) <br />
+_**Figure**. A typical AppSec security program's core workflows and the scope of out-of-the-box Semgrep OSS and Semgrep AppSec Platform features._
+
+### Deployment
+
+The process of integrating Semgrep into your developer and infrastructure workflows._
+
+<div class="col-2-grid" >
+<div> 
+
+##### Semgrep OSS
+
+Semgrep OSS runs in your local machine's CLI through the `semgrep scan` command.
+
+Deploying in bulk or at scale is manual: it can scan a remote repository by running a CI job but you must write and configure the CI job for each repository. 
+
+<!-- Environments are omitted due to differences in licensing -
+`semgrep scan` can be run in IDE atm but the extension is proprietary -->
+
+</div>
+<div>
+
+##### Semgrep AppSec Platform 
+
+Semgrep AppSec Platform can perform scans in the following environments:
+
+- CI
+- Web app (for Managed Scans)
+- CLI
+- IDE
+- `pre-commit`
+
+Your scan configuration, such as rules and policies, and scan analysis (SAST, SCA, or secrets) are preserved across all environments.
+
+Users comfortable with granting Semgrep code access, can quickly deploy Semgrep to thousands of repositories through [Managed Scans](/deployment/managed-scanning).
+
+AppSec Platform supports various CI providers and source code managers (SCMs) such as GitHub, GitLab, Bitbucket, and Azure.
+
+</div>
+</div>
+
+### Scanning and analyses
+
+_The process of analyzing source code for findings. This section explains the analyses available to both product offerings._
+
+<div class="col-2-grid"> 
+<div>
+
+##### Semgrep OSS
+
+Semgrep OSS provides the following SAST analyses:
+
+- Single file, cross function constant propagation
+- Single function taint analysis 
+- Semantic analysis
+
+The scope makes it fast, at the cost of coverage and precision.
+
+It can't track data beyond a single function or file and may find more false positives.
+
+</div>
+<div>
+
+##### Semgrep AppSec Platform 
+
+
+Semgrep AppSec Platform supports SAST, SCA, and secret scans as listed in [Product terms](#product-terms). You can run these **scan types** across all of your environments, preserving any configuration you have made.
+
+<details>
+<summary>Click to view Semgrep Code analyses (SAST)</summary>
```

Comment:
```suggestion
<summary>Click to view Semgrep Code analyses and functions (SAST)</summary>
```

Not sure if this omission was deliberate or not, but if it is, feel free to ignore this comment

Reply:
The Semgrep Code section was indeed the only product that was purely about SAST analyses - Dependency search in SCA isn't really an analysis, more a search function and Historical scans in Secrets is also a function, not an analysis.